premake5.lua

```diff
@@ -302,6 +302,16 @@ function addSourceDir(path)
         path .. "/*.hpp",       -- C++ style headers (for glslang)
         path .. "/*.natvis",    -- Visual Studio debugger visualization files
     }
+    removefiles
+    {
```

Comment:
As discussed elsewhere I find having these in the project useful. Perhaps they are only removed with an option? So can run premake on local machine.

Alternatively there is logic in premake that knows what all the extractor files are, and so we could add so they are always there, even if the file is not present. 

Anyway I guess none of this should stop this PR.